myapp/app/controllers/users_controller.rb

```diff
@@ -1,5 +1,6 @@
 class UsersController < ApplicationController
   def new
+    redirect_to tasks_path, notice: 'Already logged in!' if logged_in?
```

Comment:
same as https://github.com/Fablic/training/pull/1667/files#r1632505976

Reply:
Fixed it in eb925c6